test/groovy/org/ods/core/test/wiremock/WiremockManager.groovy

```diff
@@ -97,10 +101,26 @@ class WiremockManager {
         if (!equalToJsonField)
```

Comment:
here shoud be also jsonBuilderFromFile.content.request.bodyPatterns[0].equalToJson